src/screens/Button.js

```diff
@@ -925,6 +994,15 @@ const ButtonPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="button.disabled.secondary.icon">
```

Comment:
Remove all icon specific lines related to state since they're covered by the `button.default`, `button.secondary`, etc.

Reply:
oh yes sorry thought I pushed that up